### DISCUSSION TITLE
ECS Skeletals

### DISCUSSION THREAD

**Discussion by fire** [GodotECS/godex#226] [Ideas & Proposals]
### Describe the project you are working on
3D Game

### Describe the problem or limitation you are having in your project
Wanted better performance on skeleton animation and IK

### Describe the feature / enhancement and how it helps to overcome the problem or limitation
Define skeletons in ecs

1. skeleton
2. animation player
2. skeletal animation input to the compute shaders for skinning and blend shapes
3. ik

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

???

### If this enhancement will not be used often, can it be worked around with a few lines of script?

IK and Skeleton is thousands of lines of code.

### Is there a reason why this should be core and not an add-on in the asset library?

GDScript is slow

**Comment by AndreaCatania** [comment]
Moving the proposal to discussions to keep the issue just for things to fix.
